index.html

```diff
@@ -0,0 +1,211 @@
+                    <a href="#main-program" onclick="closeNav()">Main Program</a>
+                    <a href="#speakers" onclick="closeNav()">Speakers</a>
+                    <a href="#partners" onclick="closeNav()">Partners</a>
+                    <a href="#" onclick="closeNav()">Join the Program</a>
+                    <a href="#contact" onclick="closeNav()">Contact</a>
+                </div>
+            </div>
+            <div class="desktop-navbar">
+                <a href="./index.html" class="main-logo"><img src="./Assets/images/Stem logo.png" alt="main logo"
+                        class="main-logo-img" /></a>
+                <ul class="navbar-ul">
+                    <li><a href="./about.html" class="desktop-nav-link">About</a></li>
+                    <li>
+                        <a href="#main-program" class="desktop-nav-link">Main Program</a>
+                    </li>
+                    <li><a href="#speakers" class="desktop-nav-link">Speakers</a></li>
+                    <li><a href="#partners" class="desktop-nav-link">Partners</a></li>
+                    <li><a href="#" class="desktop-nav-link">Join the Program</a></li>
+                    <li><a href="#contact" class="desktop-nav-link">Contact</a></li>
+                </ul>
+            </div>
+        </nav>
+        <div class="container">
+            <img src="./Assets/images/website-bg.png" alt="Background image" class="bg-header-image" />
+        </div>
+        <div class="main-content">
+            <div class="main-content-headers">
+                <h2 class="special-color">"Hello! Sharing world"</h2>
+                <h1 class="special-color">
+                    JAVASCRIPT JUNCTION: UNITING DEVELOPERS FOR A CODE-INFUSED SOCIETY...
+                </h1>
+                <h3 class="year special-color heading-big">(JSOCIETY) 2023</h3>
+            </div>
+            <div class="header-content text-center">
+                <p>
+                    Join us at JavaScript Junction, a vibrant community meetup designed to bring together JavaScript
+                    enthusiasts, developers, and industry professionals. This event is a perfect platform for expanding
+                    your network, sharing ideas, and staying up-to-date with the latest trends and innovations in the
+                    JavaScript ecosystem.
+                    <br> The JSOCIETY Strategy - Knowledge Sharing: We encourage open dialogue and knowledge sharing
+                    among community members.
+                    <br> <b> SHARPENING THE FOCUS ON jAVASCRIPT.</b>
+                </p>
+            </div>
+            <div class="infos">
+                <h3 class="date light-black heading-medium">
+                    2023.06.17(MON) ~ 20(WED)
+                </h3>
+                <p class="location light-black heading-small">
+                    @Taifa Hall, University of Nairobi, Nairobi Kenya.
+                </p>
+            </div>
```

Comment:
- [ ] Please add padding to the `main-content` div in the mobile version as the content seems to be to cramped and collapsing with the border. Adding more space would make it look clean and professional.
<img width="364" alt="Screenshot 2023-07-07 at 10 25 47 AM" src="https://github.com/peter-mwovi/Capstone-project/assets/44798044/03bb57b8-ae22-47ff-abbf-3428e8fdd1d7">